clients/privacy-center/components/modals/consent-request-modal/ConsentRequestModal.tsx

```diff
@@ -31,6 +31,7 @@ export const useConsentRequestModal = () => {
     setIsOpen(false);
     setCurrentView(ModalViews.ConsentRequest);
     setConsentRequestId("");
+    router.push('/', undefined, { shallow: true });
```

Comment:
`shallow: true` seemed to be the thing to do here to tell React not to re-render anything more than it absolutely has to (for instance, I think props are still refreshed)